src/main/java/ip/syssrc/string/Vocab.java

```diff
@@ -0,0 +1,100 @@
+package ip.syssrc.string;
+
+/**
+ * Vocab
+ *
+ * Assignment 5.2
+ *
+ * @author H071211045 - Muhammad Sofyan Daud Pujas <gaero38@gmail.com>
+ *
+ */
+public class Vocab {
+
+    /**
+     * Add 'un' prefixes to a word, e.g. happy -> unhappy
+     *
+     * @param word the word to be prefixed
+     * @return string of prefixes word
+     */
+    public static String addPrefixUn(String word) {
+        String prefixUn = new String("un");
+        word = word.toLowerCase();
+
+        if (!word.startsWith(prefixUn)) {
+            word = prefixUn + word;
+        }
+        return new String(word);
+    }
+
+    /**
+     * Add prefix for each word of groups of vocabulary words
+     *
+     * @param prefix the prefix to be added to each word
+     * @param groups the groups of vocabulary words to be prefixed
+     * @return string in the form of: `prefix :: prefixword_1 :: prefixword_2 :: ...
+     *         :: prefixword_n`
+     */
+    public static String prefixes(String prefix, String... groups) {
+        String merged = "";
+        String separator = new String(" :: ");
+        int opt = 1;
+
+        for (String element : groups) {
+            merged = merged.concat(prefix).concat(element);
+
+            if (opt >= groups.length) {
+                break;
+            }
+            merged = merged.concat(separator);
+            ++opt;
```

Comment:
`foreach` *statement* digunakan jika indeks tidak dibutuhkan, variable `opt` disini
bertindak seperti indeks, jika membutuhkan kondisi yang bergantung pada indeks,
sebaiknya gunakan `for` *statement*